Source/Wikitivity.PreInstallInstanceSetting/PreInstallEventHandler.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Net;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using kCura.EventHandler;
+using kCura.EventHandler.CustomAttributes;
+using kCura.Relativity.Client;
+using Relativity.API;
+using Relativity.Services.Interfaces.InstanceSetting;
+using Relativity.Services.Interfaces.InstanceSetting.Model;
+using Relativity.Services.Objects;
+
+namespace Wikitivity.PreInstallInstanceSetting
+{
+	[RunOnce(true)]

```

Comment:
I would remove RunOnce attribute and update you code to be smart enough to query first and create the instance setting only if it doesn't already exist.